packages/stacks-classic/lib/components/input_textarea/input_textarea.less

```diff
@@ -48,15 +45,14 @@
 
     // Sizes
     &&__sm {
-        .size-styles(sm; in; @styles: fs);
-        --_in-min-h: calc(var(--su32) - var(--su4)); // 28px
-        --_in-px: calc(var(--su8) + var(--su2)); // 10px
```

Comment:
Looks like you removed the `px` variable here.  In the designs the x paddings are supposed to be 10, 11, and 13 respectively.